master/internal/resourcemanagers/resource_pool.go

```diff
@@ -344,12 +349,24 @@ func (rp *ResourcePool) Receive(ctx *actor.Context) error {
 func (rp *ResourcePool) receiveAgentMsg(ctx *actor.Context) error {
 	switch msg := ctx.Message().(type) {
 	case sproto.AddAgent:
-		ctx.Log().Infof("adding agent: %s", msg.Agent.Address().Local())
+		agentID := msg.Agent.Address().Local()
+		ctx.Log().Infof("adding agent: %s", agentID)
 		rp.agents[msg.Agent] = true
+		err := rp.updateAgentStartStats(rp.config.PoolName, agentID, msg.Slots)
+		if err != nil {
+			ctx.Log().WithError(err)
```

Comment:
does this actually log anything? probably need a `.Error` call
```suggestion
			ctx.Log().WithError(err).Error("failed to do something"
```

Reply:
done